selector-component.js

```diff
@@ -17,9 +17,11 @@ function register(module) {
       selected: '<brSelected',
       onSelect: '&brOnSelect',
       onAddItem: '&?brOnAddItem',
+      onDeselect: '&?brOnDeselect',
       fixed: '<?brFixed',
       choiceModalTitle: '@?brChoiceModalTitle',
-      showChoices: '<?brShowChoices'
+      showChoices: '<?brShowChoices',
+      allowEmpty: '<?brAllowEmpty'
```

Comment:
I don't think `brAllowEmpty` is a clear name. Also, if I interpret it as intended, we already allow this if you have no choices. Maybe: `brAllowSelectNone` to be more explicit.